app/bundles/EmailBundle/MonitoredEmail/Fetcher.php

```diff
@@ -98,9 +98,8 @@ public function fetch($limit = null)
                         $this->dispatcher->dispatch(EmailEvents::EMAIL_PARSE, $event);
                     }
 
-                    $this->log[] = $this->translator->transChoice(
+                    $this->log[] = $this->translator->trans(
                         'mautic.email.fetch.processed',
-                        $processed,
                         ['%processed%' => $processed, '%imapPath%' => $path, '%criteria%' => $criteria]
```

Comment:
```suggestion
                        ['%count%' => $processed, '%imapPath%' => $path, '%criteria%' => $criteria]
```
plus update the ini file